src/util/wallet-utils/src/getAddress.ts

```diff
@@ -72,8 +75,8 @@ export function getWalletAddresses(
       publicKey: payment.publicKey,
     },
     stacks: {
-      address: stacks.address,
-      publicKey: stacks.publicKey,
+      address: stacks?.address || "",
+      publicKey: stacks?.publicKey || "",
```

Comment:
If this is not set, are we just going to get `...` in `SBTCBalance` or something bad will happen?

Reply:
fixed